src/app/atoms/AccountTypeBadge.tsx

```diff
@@ -13,6 +13,8 @@ type AccountTypeBadgeProps = {
 const AccountTypeBadge = memo<AccountTypeBadgeProps>(({ account, darkTheme = false }) => {
   const title = getAccountBadgeTitle(account);
 
+  const isDarkTheme = darkTheme ? 'border-white text-white' : 'border-black text-black';
```

Comment:
it's not `isDarkTheme` variable. 
seems like `themeStyle` ore something like this